integration/src/integrationtests/resources/com/aws/greengrass/integrationtests/config.yaml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+services:
+    aws.greengrass.Nucleus: {}
+    aws.greengrass.clientdevices.Auth:
+        configuration:
+            deviceGroups:
+                formatVersion: "2021-03-05"
+                definitions:
+                    myThing:
+                        selectionRule: "thingName:myThing"
+                        policyName: "thingAccessPolicy"
+                policies:
+                    thingAccessPolicy:
+                        policyStatement1:
+                            statementDescription: "Allow client devices to perform all actions"
+                            operations:
+                                - "*"
+                            resources:
+                                - "*"
+    aws.greengrass.clientdevices.mqtt.Moquette: {}
+    main:
+        dependencies:
+            - aws.greengrass.Nucleus
+            - aws.greengrass.clientdevices.Auth
+            - aws.greengrass.clientdevices.mqtt.Moquette
```

Comment:
nucleus and moquette are just empty, can you just delete them? Why are they needed?